### PR TITLE
Update velero dep to 1.11.1

### PR DESCRIPTION
**Pull request by sseago**

### CODE REVIEW

**Review by shubham-pampattiwar — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,7 @@ require (
 	github.com/operator-framework/operator-lib v0.9.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.51.2
 	github.com/sirupsen/logrus v1.8.1
-	github.com/vmware-tanzu/velero v1.10.0 // TODO: Update this to a pinned version
+	github.com/vmware-tanzu/velero v1.11.1 // TODO: Update this to a pinned version
```
Comment (mateusoliveira43) [inline go.mod:18]: the TODO still applies?

Comment (kaovilai) [reply go.mod:18]: don't think so.. it's replaced by https://github.com/openshift/oadp-operator/pull/1125/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R165 anyways

Comment (sseago) [reply go.mod:18]: Yeah, I'm not sure what that TODO was really about. The replace below makes sure we're using our fork, but this line is updated to reflect what that fork maps to upstream.

Comment (kaovilai) [reply go.mod:18]: it was v000.0000000 something IIRC

Comment (sseago) [reply go.mod:18]: Probably should remove that comment on master. Not sure I'd worry about it here if tests pass.

**Pull request merged**